frontend/src/ui/page/muscles.rs

```diff
@@ -69,24 +69,21 @@ pub fn view(model: &Model, data_model: &data::Model) -> Node<Msg> {
                 Msg::ChangeInterval
             ),
             domain::Muscle::iter().map(|m| {
-                let set_volume = data_model
-                    .training_stats
-                    .stimulus_for_each_muscle_per_week
-                    .get(&domain::Muscle::id(*m))
-                    .map(|stimulus_per_muscle| {
-                        stimulus_per_muscle
-                            .iter()
-                            .filter(|(date, _)| {
-                                *date >= model.interval.first
-                                    && *date <= model.interval.last.week(Weekday::Mon).last_day()
-                            })
-                            .map(
-                                #[allow(clippy::cast_precision_loss)]
-                                |(date, stimulus)| (*date, *stimulus as f32 / 100.0),
-                            )
-                            .collect()
-                    })
-                    .unwrap_or_default();
+                #[allow(clippy::cast_precision_loss)]
+                let total_7day_set_volume = common::centered_moving_total(
+                    &data_model
+                        .training_sessions
+                        .iter()
+                        .filter_map(|(_, s)| {
+                            s.stimulus_per_muscle(&data_model.exercises)
+                                .get(&domain::Muscle::id(*m))
+                                .map(|stimulus| (s.date, *stimulus as f32 / 100.))
+                        })
+                        .collect::<Vec<_>>(),
```

Comment:
```suggestion
                #[allow(clippy::cast_precision_loss)]
                let total_7day_set_volume = common::centered_moving_total(
                    &data_model
                        .training_sessions
                        .values()
                        .filter_map(|s| {
                            s.stimulus_per_muscle(&data_model.exercises)
                                .get(&domain::Muscle::id(*m))
                                .map(|stimulus| (s.date, *stimulus as f32 / 100.))
                        })
                        .collect::<Vec<_>>(),
```

Reply:
This feels like a situation that Clippy should suggest an improvement for.